src/main/steps/terms-and-conditions/content.test.ts

```diff
@@ -7,163 +7,110 @@ jest.mock('../../app/form/validation');
-    'This online service contains several free-text fields in which you may choose to enter information which is personally sensitive. If you enter information about your:',
-  religiousBeliefs: 'religious beliefs',
-  sexualOrientation: 'sexual orientation or sex life',
-  racialOrigins: 'racial or ethnic origins',
-  politicalOpinions: 'political opinions',
-  philosophicalBeliefs: 'religious or philosophical beliefs',
-  tradeMembership: 'trade union membership',
-  geneticData: 'genetic data or biometric data',
-  healthData: 'health data',
-  consentGiven:
-    'you give us your consent to processing this information for the purpose of dealing with your application. For more information about how we collect and store your personal data, see the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
-  disclaimer: 'Disclaimer',
-  disclaimerGuarantee:
-    "While we make every effort to keep information up to date, we don't provide any guarantees, conditions or warranties that it will be:",
-  current: 'current',
-  secure: 'secure',
-  accurate: 'accurate',
-  complete: 'complete',
-  bugFree: 'free from bugs or viruses',
-  advice:
-    "We don't publish advice. You should get professional or specialist advice before doing anything on the basis of the information in the service. We're not liable for any loss or damage that may come from using this service. This includes:",
-  losses: 'any direct, indirect or consequential losses',
-  civilWrongs:
-    "any loss or damage caused by civil wrongs ('tort', including negligence), breach of contract or otherwise",
-  serviceUse: 'the use of this service, GOV.UK and any websites that are linked to or from it',
-  inabilityToUse: 'the inability to use this service, GOV.UK and any websites that are linked to or from it',
-  damageForeseeable:
-    "This applies if the loss or damage was foreseeable, arose in the normal course of things or you advised us that it might happen. This includes (but isn't limited to) the loss of your:",
-  income: 'income or revenue',
-  salary: 'salary, benefits or other payments',
-  business: 'business',
-  profits: 'profits or contracts',
-  opportunity: 'opportunity',
-  anticipatedSavings: 'anticipated savings',
-  data: 'data',
-  goodwill: 'goodwill or reputation',
-  tangibleProperty: 'tangible property',
-  intangibleProperty: 'intangible property, including loss, corruption or damage to data or any computer system',
-  wastedTime: 'wasted management or office time',
-  liable: 'We may still be liable for:',
-  death: 'death or personal injury arising from our negligence',
-  fraudulentMisrepresentation: 'fraudulent misrepresentation',
-  anyOtherLiability: 'any other liability which cannot be excluded or limited under applicable law',
-  contactUsFurtherInfo: '<a class="govuk-link" href="/contact-us">Contact us</a> for further information.',
+  thisServiceIs: 'This Service is managed by Her Majesty’s Courts and Tribunals service.',
```

Comment:
Her Majesty's? Dont think that's appropriate now. Put this as HM for now

Reply:
changed to hm